docs/docsite/rst/os_guide/windows_app_control.rst

```diff
@@ -0,0 +1,153 @@
+
+When setting up a policy it is recommended to configure Ansible as a supplemental policy so it can be easily modified and applied where Ansible will be used. Whether you use a supplemental or just a base policy for trusting the certificate used by Ansible, the base policy must have the following options set:
+
+* User Mode Code Integrity (``0 Enabled:UMCI``) is enabled
+* Disable Script Enforcement (``11 Disabled:Script Enforcement``) is not enabled
+* Dynamic Code Security (``19 Enabled:Dynamic Code Security``) is enabled
+
+The policy then should then add the certificate as a trusted publisher to the ``User Mode Signing Scenario``, for example this is an example policy configuration that contains a trusted publisher:
+
+.. code-block:: text
+
+   <SiPolicy>
+      ...
+      <Signers>
+         <Signer Name="Some Signer" ID="ID_SIGNER_S_0">
+            <CertRoot Type="TBS" Value="1DBF60AFC6313593EDB09B6C6239BE493FF3461D4BD6D0A8C6E1723A12C06438F471BB7F6BAA73BD142D0698CEFF9DBB" />
+            <CertPublisher Value="Some Publisher" />
+         </Signer>
+      </Signers>
+      <SigningScenarios>
+         <SigningScenario ID="ID_SIGNINGSCENARIO_KMCI" FriendlyName="Kernel Mode Signing Scenario" Value="131">
+            <ProductSigners />
+         </SigningScenario>
+         <SigningScenario ID="ID_SIGNINGSCENARIO_UMCI" FriendlyName="User Mode Signing Scenario" Value="12">
+            <ProductSigners>
+            <AllowedSigners>
+               <AllowedSigner SignerId="ID_SIGNER_S_0" />
+            </AllowedSigners>
+            </ProductSigners>
+         </SigningScenario>
+      </SigningScenarios>
+      ...
+   </SiPolicy>
+
+Once the policy is created and the certificate that will be used to sign the Ansible content is trusted by the Windows host, the policy can be applied.
+
+.. Warning::
+     As Ansible typically runs tasks as an Administrator, it is important that the policy is signed and is applied so that Ansible cannot unset the policy through a task like ``win_file`` or ``win_regedit``.
+
+How to Sign Ansible Content
+---------------------------
+Once the code signing certificate has been generated and trusted by the Windows host, it can be used to sign the scripts that Ansible will run. The PowerShell script `New-AnsiblePowerShellSignature.ps1 <https://raw.githubusercontent.com/ansible/ansible-documentation/refs/heads/devel/examples/scripts/New-AnsiblePowerShellSignature.ps1>`_ can be used to sign both the execution wrapper used by Ansible to invoke modules and any PowerShell modules inside an Ansible collection. It requires the following to run:
+
+* PowerShell 7.4 or later
+* The `OpenAuthenticode <https://github.com/jborean93/PowerShell-OpenAuthenticode>`_ PowerShell module
```

Comment:
Just going to put this comment here for consideration. Should this repo move to the Ansible org so it is "more official" than a personal repo?